lib/getMermaidDiagram.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * Generates mermaid class diagram from default syntax
+ * 
+ * @param {Object} metrics Relations between AsyncAPI docs in default syntax
```

Comment:
I think that we should change the name of `syntax` to `output`. What do you think? 🤔

Reply:
I think the output we are providing should be like `default output and the formats we are supporting should be like `mermaid syntax`, `plantUML syntax` etc...

Reply:
Of course, I agree, but I mean that we should not write about output as `syntax`, or we should change it to `output syntax`, `default output syntax` etc so as not to be misleading.

Reply:
`default output syntax` would be good 👍🏽